ports/servoshell/minibrowser.rs

```diff
@@ -78,62 +130,127 @@ impl Minibrowser {
             context,
             event_queue,
             toolbar_height,
+            widget_surface_fbo,
             last_update,
             location,
             location_dirty,
+            ..
         } = self;
+        let our_fbo = *widget_surface_fbo;
```

Comment:
```suggestion
        let widget_fbo = *widget_surface_fbo;
```

Reply:
done in f6f576b20cbdafa4765c66ac400894be392caa91